code/src/App.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import { Home } from 'pages/Home'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div >
+      <Home />  
     </div>
```

Comment:
Can't see that this div is styled somewhere, so you can remove it, and just mount `<Home/>`.